core/src/main/java/org/openstack4j/openstack/telemetry/internal/BaseTelemetryAodhServices.java

```diff
@@ -0,0 +1,36 @@
+package org.openstack4j.openstack.telemetry.internal;
+
+import com.google.common.base.Function;
+import com.google.common.collect.Lists;
+import org.openstack4j.api.types.ServiceType;
+import org.openstack4j.openstack.internal.BaseOpenStackService;
+
+import java.util.Collections;
+import java.util.List;
+
+public class BaseTelemetryAodhServices extends BaseOpenStackService {
+
+	protected BaseTelemetryAodhServices() {
+		super(ServiceType.TELEMETRY_AODH, EndpointFunction.instance);
+	}
+	
+
+	private static class EndpointFunction implements Function<String, String> {
```

Comment:
thank for your suggestion,path constant maybe a good idea at present.but,telemetry service endpoint may be changed with version information or not in the future. EndpointFunction solution can work with that situation.  another solution we can do like
![image](https://cloud.githubusercontent.com/assets/18585129/21339928/e45f5e8c-c6bd-11e6-9cb6-de5e86f00fd6.png)

what is your idea,auhlig?